prime-dotnet-webapi/Models/FacilityCode.cs

```diff
@@ -0,0 +1,11 @@
+namespace Prime.Models
+{
+    public enum FacilityCode
+    {
+        AcuteAmbulatoryCare = 1,
+        LongTermCare,
```

Comment:
I would prefer seeing all of these being explicitly assigned rather than inferred